console/src/main/webapp/manager/app/app.es6

```diff
@@ -1,7 +1,7 @@
 class AppController {
-  static $inject = [ '$scope', '$router', '$location' ]
+  static $inject = [ '$scope', '$router', '$location' , 'roleAdminList']
```

Comment:
extra space before comma

Reply:
missing space before ]

Reply:
`npm run lint` should catch all of these for you